index.js

```diff
@@ -689,6 +696,47 @@ class Hyperbee extends ReadyResource {
     return (this._checkout <= this.core.length || this._checkout <= 1) ? this.core.snapshot() : this.core.session({ snapshot: false })
   }
 
+  async clearUnlinked (options = {}) {
```

Comment:
Sanity check: is it safe to run this while running other operations on the bee (like putting and deleting blocks)?

Reply:
Should be safe since it uses checkouts below